lib.rs

```diff
@@ -314,7 +315,7 @@ mod pe {
 
             let resource_size = SizeofResource(current_process_hmod, resource_handle);
             if resource_size == 0 {
-                return Err(std::io::Error::last_os_error());
+                return Ok(Some(&[]));
```

Comment:
Noticed this while debugging. I accidentally did this here: https://github.com/denoland/sui/pull/37/files#diff-0544fc952eefa8d170bbd200b75827f5bb58e39b132e7c871c1bc8a9d5e09ce5R317